api/indexer/client.go

```diff
@@ -53,8 +56,8 @@ func (c *Client) GetBlockByHeight(ctx context.Context, height uint64, parser cha
 	resp := GetBlockClientResponse{}
 	err := c.requester.SendRequest(
 		ctx,
-		"getBlockByHeight",
-		&GetBlockByHeightRequest{Height: height},
+		"getBlock",
```

Comment:
Out of scope for this PR - we should move the parser from an argument to each function to an argument to construct the client instance, so that we don't need to pass it around each time